projects/schematics/src/add-spartacus/schema.json

```
@@ -184,6 +185,10 @@
             "value": "Product-Variants",
             "label": "Product - Variants"
           },
+          {
+            "value": "Product-multi-dimensional-selector",
+            "label": "Product multi-dimensional selector"
```

What do you think about changing label to
```suggestion
            "label": "Product - Multi-Dimensional Selector"
```
or
```suggestion
            "label": "Product Multi-Dimensional - Selector"
```
to keep the convention we have for other features. I like the first option because because this feature applies to the product, but I don't have a strong opinion on this.

---

thanks, I think the second one is better, as this is not a product sub-lib, but a sub-lib of product multi-dmiensional.